Week1-Coding-Task.hs

```diff
@@ -0,0 +1,76 @@
+--Section 1, credit card validation
+--Double value of every second digit from beginning of right 
+--Add digits of doubled and undoubled values to form new number 
+--Calculate result mod 10: if 0, valid 
+
+------------------------------------------------------------------------------
+--Exercise 1: toDigits and toDigitsRev 
+------------------------------------------------------------------------------
+toDigits :: Integer -> [Integer]
+toDigitsRev :: Integer -> [Integer]
+
+--A list with elements reversed, so can use it later for reversing toDigits 
+rev :: [Integer] -> [Integer]
+rev [] = []--base case 
```

Comment:
According to style-guide in our company, comments at the end of line should start after 2 spaces of code. So it should be

```haskell
rev [] = []  -- base case 
```

I think you can start following style guide in your home tasks so it will be easier for you in future when you start working on projects.